src/main/java/me/minidigger/hangar/controller/ReviewsController.java

```diff
@@ -1,21 +1,34 @@
 package me.minidigger.hangar.controller;
 
+import me.minidigger.hangar.model.NamedPermission;
+import me.minidigger.hangar.model.viewhelpers.VersionData;
+import me.minidigger.hangar.security.annotations.GlobalPermission;
+import me.minidigger.hangar.service.VersionService;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.security.access.annotation.Secured;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.servlet.ModelAndView;
 
-import me.minidigger.hangar.controller.HangarController;
-
 @Controller
 public class ReviewsController extends HangarController {
 
+    private final VersionService versionService;
+
+    @Autowired
```

Comment:
`@Autowired` isn't needed anymore to inject stuff through the constructor. For newer implementations, this should be avoided.

Reply:
I know its not strictly required anymore, why should it be avoided tho? I think it adds more clarity, just like @ Override for example